code/modules/modular_computers/file_system/programs/security/camera.dm

```diff
@@ -25,12 +25,14 @@
 	filename = "cammon"
 	filedesc = "Camera Monitoring"
 	nanomodule_path = /datum/nano_module/camera_monitor
-	program_icon_state = "generic"
+	program_icon_state = "cameras"
 	extended_desc = "This program allows remote access to station's camera system. Some camera networks may have additional access requirements."
 	size = 12
 	available_on_ntnet = 1
 	requires_ntnet = 1
-	required_access_download = access_heads
+	required_access_download = list(access_heads, access_security)
```

Comment:
Sec should have consoles with a camera preset already.
And if they want to have them on their personal devices they should contact a head of staff as anyone else would do.